packages/ni.measurementlink.measurement.v2.proto/src/ni/measurementlink/measurement/v2/measurement_service_pb2/__init__.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: ni/measurementlink/measurement/v2/measurement_service.proto
+"""Generated protocol buffer code."""
+from google.protobuf.internal import builder as _builder
+from google.protobuf import descriptor as _descriptor
+from google.protobuf import descriptor_pool as _descriptor_pool
+from google.protobuf import symbol_database as _symbol_database
+# @@protoc_insertion_point(imports)
+
+_sym_db = _symbol_database.Default()
+
+
+from google.protobuf import any_pb2 as google_dot_protobuf_dot_any__pb2
+from google.protobuf import type_pb2 as google_dot_protobuf_dot_type__pb2
+from ni.measurementlink import pin_map_context_pb2 as ni_dot_measurementlink_dot_pin__map__context__pb2
```

Comment:
Don't you need to package pin_map_context first?

Reply:
I think it's easier to do things in the right order, but if it's quicker for you to get it done this way, then go ahead.